src/main/java/org/terasology/hunger/HungerAuthoritySystem.java

```diff
@@ -86,13 +109,25 @@ public void onPlayerSpawn(OnPlayerSpawnedEvent event, EntityRef player,
         player.saveComponent(hunger);
     }
 
+    /**
+     * Saves data for a component before it's being deactivated.
```

Comment:
please be a bit more precise: "saves an entity's HungerComponent before it is deactived"

also: please no abbreviations 😉